src/import-data/protected-data-utils.ts

```diff
@@ -3,7 +3,7 @@ import { isGoogleWorkspace, WorkspaceWrapper } from 'src/libs/workspace-utils';
 
 // This method identifies whether an import source is considered protected data;
 // For now this means pfb imports from AnVIL or Biodata Catalyst.
-export const isProtectedSource = (url: string, filetype: string): boolean => {
+export const isProtectedSource = (url: string, filetype?: string): boolean => {
```

Comment:
Discovered an issue that `filetype` may be undefined in the case of BagIt imports.

Reply:
Can you add a test case for this, e.g. to `nonProtectedUrls`?

Reply:
Added in https://github.com/DataBiosphere/terra-ui/pull/4293/commits/f9d5e4b217042b1b078d61de092dcbf03735306b.